agent/bench-scripts/postprocess/linpack-prepare-input-file

```diff
@@ -226,6 +226,9 @@ echo "linpack cmd: ${cmd}"
 echo
 
 echo "exec ${cmd} | stdbuf --input=0 tee -i ${linpack_output_file}" >> "${output_dir}${linpack_script_file}"
+echo "ret=$?" >> "${output_dir}${linpack_script_file}"
```

Comment:
True, on the other hand nobody was and is checking the return code, so perhaps I can just remove the handling and leave it to the one implementing it (if needed).

Reply:
I wouldn't waste any effort on returning a value which is bad, incorrect, or misleading.  So, if you want to return a value here, it should presumably represent the result of the benchmark (or, at least, it should indicate a failure if one occurred in the pipeline); otherwise, I wouldn't bother adding special handling (if, as you say, nothing is looking for it).